packages/caliper-samples/src/contract/scalardl/src/main/java/com/example/contract/smallbank/Amalgamate.java

```diff
@@ -0,0 +1,46 @@
+package com.example.contract.smallbank;
+
+import com.scalar.ledger.asset.Asset;
+import com.scalar.ledger.contract.Contract;
+import com.scalar.ledger.exception.ContractContextException;
+import com.scalar.ledger.ledger.Ledger;
+import java.util.Optional;
+import javax.json.Json;
+import javax.json.JsonObject;
+import javax.json.JsonObjectBuilder;
+
+public class Amalgamate extends Contract {
+
+  @Override
+  public JsonObject invoke(Ledger ledger, JsonObject argument, Optional<JsonObject> properties) {
+    String srcCustomerId = "" + argument.getInt(Const.KEY_SRC_CUSTOMER_ID);
```

Comment:
It is usually better to use `Integer.toString` for such case.

Reply:
Done.